code/modules/mining/mine_items.dm

```diff
@@ -767,96 +767,6 @@
 					shake_camera(L, 20, 1)
 					addtimer(CALLBACK(L, TYPE_PROC_REF(/mob/living/carbon/human, vomit), 20))
 
-/**********************Lazarus Injector**********************/
-
-/obj/item/lazarus_injector
```

Comment:
Do not remove the item itself. 

Removing it from mining vendor is fine so it is not a common everyday item miners can just buy every round. But it should not be removed from the game or the away site completely - it could basically just be a xenoarch artifact or something.